### DISCUSSION TITLE
Force DA (difficulty adjust)  settings when in DT / HT

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#13817] [Ideas] [answered]
Was unsure of whether to put this in bugs since it isnt really one so ill put it here

Currently difficulty adjust does not counter the increase or decrease in hit windows and AR ms due to HT or DT　being applied. this leads to ar11 not being possible with HT  ( it becomes  ar10.2 ish ) or DT  with DA on ar11 ending up in some impossible to react to ar11+ region ( which leads to unnecesary guesswork to reach the desired AR ) . my suggestion would be either countering the increased and decreased AR and OD (accuracy in lazer)  windows in DA or globally countering OD and AR changing with speed mods (like mania does)

**Comment by smoogipoo** [comment, marked as answer]
Currently mania in osu!lazer does not counter OD/AR like stable does. This is related to https://github.com/ppy/osu/discussions/13187.

I think that this could be a setting, but keep in mind that DT and AR are fundamentally different concepts. DT affects the entire timing of the game for _all_ animations along with hitwindows/etc, whereas AR has a very specific domain upon which it operates and does not include many animations.
If you pixel peep reeeeeally closely, you'll see small animation differences in AR10+DT vs AR11 (via DA mod).

**Reply by mk56-spn** [reply]
thats really interesting smoogi i  didnt know that ar 11 and DT "ar11" werent identical.

sidenote. would the ability to see you accuracy on different ODs at the end of a play ever be considered? . this would help accurately compare a hard rock and  no mod play in terms of raw accuracy, this could also be applied to CS to see how many hits were within a certain constraint. although im not sure if the way osu saves replay data would allow for post match OD switching

**Reply by smoogipoo** [reply]
> would the ability to see you accuracy on different ODs at the end of a play ever be considered

I don't see why not. The original results screen statistics design has a graph of hit errors, so it could be added somewhere around there:
![image](https://user-images.githubusercontent.com/1329837/124792914-6fd97280-df88-11eb-803a-d67ee469047b.png)

**Reply by mk56-spn** [reply]
thanks for the explanation!!
